packages/amplication-client/src/Settings/NewApiToken.tsx

```diff
@@ -101,11 +101,12 @@ const NewApiToken = ({ onCompleted }: Props) => {
       </div>
       <Formik
         initialValues={INITIAL_VALUES}
+        validateOnBlur={false}
```

Comment:
I think validation on blur is a better UX, but:
1.  I guess you did it for a reason. I would like to know why 
2. maybe we don't want this behavior, but I think it should be consistent, meaning that if on our other forms we use validateOnBlur={true}, it should behave the same here

Reply:
I have made this change in order to run the validation of Formik only when a user clicks on submit button.

Not before that. So that if someone is clicking on 'X' or outside the text field without submitting it will not run to validate the input.

It will only validate the input when someone clicks on submit button. If the req. parameters are passed it will create an API token, otherwise not.

Reply:
I understood the behavior, the question if we want this behavior or we want also validation on blur. 
As a matter of fact, this form is just one input so it probably doesn't really matter

Reply:
how do you want me to continue? @abrl91

Reply:
I think you can leave it like that, but let's wait for @yuval-hazaz comment :)

Reply:
@abrl91 Please approach @yuval-hazaz in private for his response, I want to merge this pr asap.

Reply:
OK. You can confirm this PR. You are also a reviewer :) @tupe12334